src/styles/calendar.scss

```diff
@@ -0,0 +1,48 @@
+body {
+  margin: 0;
+  font-family: Arial, sans-serif;
+}
+
+@each $day, $n in $days-of-week {
+  .calendar--start-#{$day} .calendar__day:first-child {
+    margin-left: ($calendar-day-width + (3 * $day-gap)) * $n;
```

Comment:
The calculation of the margin-left is not correct, it should be multiplied by ($n - 1) to account for the first day of the week starting from 0.